content/en/docs/languages/php/sdk.md

```diff
@@ -106,14 +121,15 @@ details, see the
-| `OTEL_PHP_TRACES_PROCESSOR`          | `batch`       | `batch`, `simple`                                                                     | `simple`         | Span processor selection                                                           |
-| `OTEL_PHP_DETECTORS`                 | `all`         | `env`, `host`, `os`, `process`, `process_runtime`, `sdk`, `sdk_provided`, `container` | `env,os,process` | Resource detector selection                                                        |
-| `OTEL_PHP_AUTOLOAD_ENABLED`          | `false`       | `true`, `false`                                                                       | `true`           | Enable/disable SDK autoloading                                                     |
-| `OTEL_PHP_LOG_DESTINATION`           | `default`     | `error_log`, `stderr`, `stdout`, `psr3`, `none`                                       | `stderr`         | Where internal errors and warnings will be sent                                    |
-| `OTEL_PHP_INTERNAL_METRICS_ENABLED`  | `false`       | `true`, `false`                                                                       | `true`           | Whether the SDK should emit metrics about its internal state (eg batch processors) |
-| `OTEL_PHP_DISABLED_INSTRUMENTATIONS` | `[]`          | Instrumentation name(s), or `all`                                                     | `psr15,psr18`    | Disable one or more installed auto-instrumentations                                |
+| Name                                 | Default value | Values                                                                                | Example                      | Description                                                                        |
+| ------------------------------------ | ------------- | ------------------------------------------------------------------------------------- | ---------------------------- | ---------------------------------------------------------------------------------- |
+| `OTEL_PHP_TRACES_PROCESSOR`          | `batch`       | `batch`, `simple`                                                                     | `simple`                     | Span processor selection                                                           |
+| `OTEL_PHP_DETECTORS`                 | `all`         | `env`, `host`, `os`, `process`, `process_runtime`, `sdk`, `sdk_provided`, `container` | `env,os,process`             | Resource detector selection                                                        |
+| `OTEL_PHP_AUTOLOAD_ENABLED`          | `false`       | `true`, `false`                                                                       | `true`                       | Enable/disable SDK autoloading                                                     |
```

Comment:
Could you add a bit more detail in the descriptions as to what users can accomplish by fiddling with these? In a separate PR, if you prefer.

Reply:
Yes, I'll do this in another PR - it might need some reworking.

Reply:
done, #3959